test/e2e/specs/editor/local/demo.spec.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'New editor state', () => {
+	test( 'content should load, making the post dirty', async ( {
+		page,
+		admin,
+	} ) => {
+		await admin.visitAdminPage( 'post-new.php', 'gutenberg-demo' );
+		await page.waitForFunction( () => {
+			if ( ! window?.wp?.data?.dispatch ) {
+				return false;
+			}
+			window.wp.data
+				.dispatch( 'core/preferences' )
+				.set( 'core/edit-post', 'welcomeGuide', false );
+
+			window.wp.data
+				.dispatch( 'core/preferences' )
+				.set( 'core/edit-post', 'fullscreenMode', false );
+
+			return true;
+		} );
```

Comment:
Would it make sense rather to extend the `createNewPost` util? For example:

```js
export async function createNewPost( {
	postType,
	title,
	content,
	excerpt,
	customQuery,
	showWelcomeGuide = false,
} = {} ) {
	const query = addQueryArgs( '', {
		post_type: postType,
		post_title: title,
		content,
		excerpt,
		...customQuery,
	} ).slice( 1 );
	// ...
```

And then in the test:

```js
await admin.createNewPost( {
	customQuery: { 'gutenberg-demo': null },
} );
```

Reply:
It's probably not worth it for a single use case, and this duplication isn't that massive :)